cypress/e2e/movie_details_spec.cy.js

```diff
@@ -0,0 +1,29 @@
+describe('empty spec', () => {
```

Comment:
@zacwalters4 hey! Is this suppose to say `empty spec` or is that the name of the test?

Reply:
Fixed thank you!